vanoce.php

```diff
@@ -87,6 +85,47 @@
         <strong>Celkem dárků: <?php echo count($darky);?></strong>
     </p>
 
+    <!-- úkol 2. -->
+    <h2>Příliš drahé dárky</h2>
+    <ul class="list-group my-3">
+        <?php foreach ($darky as $darek) {
+            if ($darek["cena"] > 1000) { ?>
+                <li class="list-group-item"><?php echo $darek["popis"]; ?></li>
+            <?php }
+        } ?>
+    </ul>
+
+    <!-- úkol 3. -->
+    <h2>Etikety</h2>
+    <ul class="list-group my-3">
+        <?php foreach ($darky as $darek) { ?>
+            <div class="card mt-1 w-25">
+                <div class="card-body display-3">
+                    <?php echo $darek["jmeno"]; ?>
+                </div>
+            </div>
+        <?php } ?>
+    </ul>
+
+    <!-- úkol 4. -->
+    <h2>Etikety</h2>
```

Comment:
```suggestion
    <h2  class="mt-2">Etikety (s přizpůsobením velikosti písma)</h2>
```

at je jasny, ze to je ten dalsi ukol